### DISCUSSION TITLE
Add Right Click Open New Connection on Console Tab.

### DISCUSSION THREAD

**Discussion by gtdegamo** [dbeaver/dbeaver#14452] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

Hello.  I have a long list of connections in my connections panel.  And often for a connection, I'll expand out the schema list and tables/procs.  Often times, I need to open a new console with the same connection as the existing console tab  that I am on.  So, what I do now, is go to my connection panel, and scroll down to the the connection, click the connection, and the right click to open new connection.  It's really time consuming, especially if you have the connection panel minimized, and/or you have many items expanded in your connection panel.

**Describe the solution you'd like**

What would be nice is to right click your current console tab.  And open new connection.  This will create a new tab having the same connection as the current console tab.

**Describe alternatives you've considered**

Currently, I have to scroll through my list of connections on the connection panel to find the right one.

**Additional context**

None.

**Comment by Matvey16** [comment, marked as answer]
@gtdegamo You can open another console for the same connection with menu buttons. While current console is in focus click `SQL Editor -> Open SQL Console`

![RlVCwpwpFM](https://user-images.githubusercontent.com/82543000/136918991-8f9c4baa-66db-4e01-b8d8-1429ea7adc62.gif)
